cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/init/models/SrcFile.java

```diff
@@ -147,23 +147,23 @@
 
     private FileType srcFileType;
     private String content;
-    private String testFileContent;
     private String name;
-    private String testFileName;
     public SrcFile(String name, FileType fileType) {
         this.srcFileType = fileType;
         this.name = name;
         switch (fileType) {
```

Comment:
this kind of thing was ok when we had only two types, since we have many now, we need to refactor to be more OOP, switch cases are anti-oop

Reply:
Fixed with 3dd1cb2